### ISSUE TITLE
Waypoints fails to load with dynmap

### ISSUE THREAD

**Issue by gazpachoking** [Sytm/waypoints2#58]
Aww crap, I got really excited for waypoints on my map, but having a bit of an issue.

**Describe the bug**
Crash when loading with Dynmap installed.

**To Reproduce**
Start server with Dynmap 3.3-beta-1, and Waypoints 3.2.0 on paper 1.18.1 build 69

**Expected behavior**
Waypoints to load.

**Screenshots**

**Additional context**
```
[23:31:44 ERROR]: Error occurred while enabling Waypoints v3.2.0 (Is it up to date?)
java.lang.NullPointerException: markerApi.getMarkerIcon(…integrations.dynmap.icon) must not be null
        at de.md5lukas.waypoints.integrations.DynMapIntegration.setupDynMap(DynMapIntegration.kt:37) ~[Waypoints-LATEST-[v3.2.0].jar:?]
        at de.md5lukas.waypoints.WaypointsPlugin.initIntegrations(WaypointsPlugin.kt:154) ~[Waypoints-LATEST-[v3.2.0].jar:?]
        at de.md5lukas.waypoints.WaypointsPlugin.onEnable(WaypointsPlugin.kt:74) ~[Waypoints-LATEST-[v3.2.0].jar:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:500) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_18_R1.CraftServer.enablePlugin(CraftServer.java:563) ~[paper-1.18.1.jar:git-Paper-69]
        at org.bukkit.craftbukkit.v1_18_R1.CraftServer.enablePlugins(CraftServer.java:477) ~[paper-1.18.1.jar:git-Paper-69]
        at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:727) ~[paper-1.18.1.jar:git-Paper-69]
        at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:503) ~[paper-1.18.1.jar:git-Paper-69]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:313) ~[paper-1.18.1.jar:git-Paper-69]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1202) ~[paper-1.18.1.jar:git-Paper-69]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:317) ~[paper-1.18.1.jar:git-Paper-69]
        at java.lang.Thread.run(Thread.java:833) ~[?:?
```

- [x] I am using the latest version of Waypoints available

**Comment by gazpachoking** [comment]
Tried adding this to my config (which didn't get added automatically, even when deleting my config), but I still can't seem to get it to work, even though that seems like it should be the issue.
```
integrations:
  dynmap:
    # See https://github.com/webbukkit/dynmap/wiki/Using-Markers#marker-icons for more information
    icon: default   # tried 'compass' too, which seems more apropos
```

**Comment by SwierczKacper** [comment]
Same here.

Log: https://pastebin.com/a78NdFq4

**Comment by Sytm** [comment]
Sorry guys, small copy and paste mistake. Has been fixed in 3.2.1
